src/cmd/sql_cmd_test.cc

```diff
@@ -309,13 +309,6 @@ TEST_P(DBSDKTest, create_without_index_col) {
     ASSERT_TRUE(cs->GetNsClient()->DropTable("test2", "trans", msg));
 }
 
-TEST_P(DBSDKTest, show) {
```

Comment:
why rm this case ?

Reply:
not rm, cases about global variable are removed into sql_cluster_test